metabase/metabase
# Mantine Select

Closes https://github.com/metabase/metabase/issues/33278

How to verify:
- `yarn storybook`
- Open http://localhost:6006/?path=/docs/inputs-select--default-story
- Compare with https://www.figma.com/file/21uCY0czmCfb6QBjRce0I8/Input-%2F-Select?type=design&node-id=1-96&mode=design&t=ymlACv8TfcJGYFGM-0

Notes:
- There is no story for disabled items since there is always a disabled item.
- There is no story for a custom item since we actually set our own custom item component by default. In other words, it's always custom under the hood. You can check `SelectItem` to see how it's implemented.

<img width="1092" alt="Screenshot 2023-08-29 at 17 53 10" src="https://github.com/metabase/metabase/assets/8542534/74be202d-cd3a-456f-8384-e053b34c3435">
